drivers/flash/flash_stm32_ospi.c

```diff
@@ -50,6 +51,27 @@ LOG_MODULE_REGISTER(flash_stm32_ospi, CONFIG_FLASH_LOG_LEVEL);
 #define SPI_NOR_WRITEOC_NONE 0xFF
 
 #if STM32_OSPI_USE_DMA
+#if CONFIG_DT_HAS_ST_STM32U5_DMA_ENABLED
```

Comment:
We can't use this symbol. Its usage is restricted.
We should use DMA_STM32U5

Reply:
using CONFIG_DMA_STM32U5